classwork/intro-extra-workflowsets-classwork.qmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Extra - Workflowsets - Classwork"
+subtitle: "Introduction to tidymodels"
+editor_options: 
+  chunk_output_type: console
+---
+
+We recommend restarting R between each slide deck!
+
+## Setup
+
+Setup
+
+```{r}
+library(tidymodels)
+library(forested)
+
+set.seed(123)
+forested_split <- initial_split(forested, prop = 0.8)
+forested_train <- training(forested_split)
+forested_test <- testing(forested_split)
+
+set.seed(123)
+forested_folds <- vfold_cv(forested_train, v = 10)
+
+tree_spec <- decision_tree(mode = "classification")
```

Comment:
```suggestion
tree_spec <- decision_tree(cost_complexity = 0.0001, mode = "classification")
```

Just to align with the other classworks.

Reply:
It wasn't in the deck but I've decided to add it in both places :)